scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -197,16 +286,32 @@ adt_df <- adt_df %>%
   )
 ```
 
-```{r}
+Now we will want to add the cell types we have assigned back to our original SCE object.
+We can do that by defining a new column name, `threshold_celltype` that will be added to the `colData` object.
+Creating and assigning values to this column can be done with the `$` shortcut, and then we can plot our results with the `plotUMAP()` function as before.
+
+```{r plot thresholds}
```

Comment:
Should point out that even though this is ADT information we still are looking at our RNA-level UMAP! So, this data indeed goes into the main experiment coldata, not the ADT experiment coldata.